source/core/src/main/com/csse3200/game/components/tasks/PauseTask.java

```diff
@@ -92,6 +96,11 @@ public void update() {
             // NPC pauses when close enough to the target
             hasApproached = true;
             movementTask.stop();
+            logger.info("Pausing");
```

Comment:
Can we set this to debug - there'll be a lot of these popping up if anyone wants to do functional testing.

Reply:
This is just being removed - I forgot about it sorry!